admin-ui/app/components/GluuDropdown/GluuDropdown.tsx

```
@@ -0,0 +1,233 @@
+          onClick={() => handleOptionClick(option)}
+        >
+          {option.icon && <span>{option.icon}</span>}
+          {optionContent}
+        </div>
+      )
+    })
+  }, [filteredOptions, isSelected, renderOption, classes, emptyMessage, handleOptionClick])
+
+  return (
+    <div className={`${classes.dropdownWrapper} ${className || ''}`} ref={dropdownRef}>
+      <div onClick={handleTriggerClick} style={{ cursor: disabled ? 'not-allowed' : 'pointer' }}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)